README.md

```diff
@@ -38,35 +34,39 @@ This repo includes a fully functional example app featuring threads, reactions,
+The official documentation for the Chat SDK is available [on our website](https://getstream.io/chat/docs/android/?language=kotlin). Each feature's page shows how to use it with the Android SDK, plus there are further Android-exclusive docs under the Android sections on the top.
 
-The Chat Android SDKs support both Kotlin and Java usage, but *we strongly recommend using Kotlin*. The documentation is available in both languages - see [here](https://getstream.io/chat/docs/?language=java) for the Java version.
+The Chat Android SDKs support both Kotlin and Java usage, but *we strongly recommend using Kotlin*. The documentation is available in both languages.
 
 You can also take a look at the [full API documentation](https://getstream.github.io/stream-chat-android/) (generated by Dokka).
 
 This SDK consists of the following modules / artifacts:
 
-- [Chat client](stream-chat-android-client)
-- [Offline support and `LiveData` APIs](stream-chat-android-offline)
-- [Chat UI/UX](stream-chat-android)
+- [Chat client](/stream-chat-android-client)
+- [Offline support and `LiveData` APIs](/stream-chat-android-offline)
+- [Chat UI/UX](/stream-chat-android-ui-components)
 
 With these modules, the SDK provides:
 
 - A low-level client for making API calls and receiving chat events
 - Offline support and LiveData APIs module
 - Ready to use ViewModels for displaying a list of channels and a conversation 
 - Reusable chat views:
-    - [Channel List](https://getstream.io/chat/docs/channels_view/?language=kotlin)
-    - [Message List](https://getstream.io/chat/docs/message_list_view/?language=kotlin)
-    - [Message Input](https://getstream.io/chat/docs/message_input_view/?language=kotlin)
-    - [Channel Header](https://getstream.io/chat/docs/channel_header_view/?language=kotlin)
-    - [Message Input View](https://getstream.io/chat/docs/message_input_view/?language=kotlin)
-
+    - [Channel List View](https://getstream.io/chat/docs/android/channel_list_view/?language=kotlin)
+    - [Channel List Header View](https://getstream.io/chat/docs/android/channel_list_header_view/?language=kotlin)
+    - [Message List View](https://getstream.io/chat/docs/android/message_list_view/?language=kotlin)
+    - [Message List Header View](https://getstream.io/chat/docs/android/message_list_header_view/?language=kotlin)
+    - [Message Input View](https://getstream.io/chat/docs/android/message_input_view/?language=kotlin)
+    - [Search Input View](https://getstream.io/chat/docs/android/search_input_view/?language=kotlin)
+    - [Search Result List View](https://getstream.io/chat/docs/android/search_result_list_view/?language=kotlin)
+    - [Attachment Gallery](https://getstream.io/chat/docs/android/attachmentgallery/?language=kotlin)
```

Comment:
We also have `SuggestionListView` which should be connected with `MessageInputView`, but we don't mention about that anywhere

Reply:
Creating a Jira task for this